Android/app/src/main/java/tie/hackathon/travelguide/destinations/description/FinalCityInfo.java

```diff
@@ -47,7 +39,6 @@
     private String id;
     private String tit;
```

Comment:
These were already present, it definitely hit my mind to change these names but wanted to re-use the old code as much as possible. I'll change them.